src/style.css

```diff
@@ -0,0 +1,67 @@
+body {
+  margin: 0;
+  font-family: Arial, sans-serif;
+}
+
+.container {
+  display: grid;
+  grid-template-areas: "b1 b2 b3" "b6 b5 b4";
```

Comment:
you need to make this task with flex, but save this solution it will come in handy in the future